src/js/helpers/Util.js

```diff
@@ -154,6 +157,34 @@ var Util = {
   },
   getUniqueId: function () {
     return ++consecutiveNumber;
+  },
+  detectPathsInObject: function (obj, startKey, excludePaths = []) {
```

Comment:
Yes, I am also a bit unhappy with the naming. But convertToObjectPath doesn't fit too.
This functions returns multiple paths as an array and there is no convertion, so I would let the name like it is until we find a better name.